x-pack/plugins/reporting/server/usage/decorate_range_stats.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CSV_JOB_TYPE, PDF_JOB_TYPE, PNG_JOB_TYPE } from '../../common/constants';
+import { AvailableTotal, FeatureAvailabilityMap, RangeStats, ReportingFeature } from './';
+
+function getForFeature(
+  range: Partial<RangeStats>,
+  featureKey: ReportingFeature,
+  featureAvailability: FeatureAvailabilityMap,
+  additional?: any
+): AvailableTotal & typeof additional {
+  const isAvailable = (feature: ReportingFeature) =>
+    featureAvailability[feature] != null ? featureAvailability[feature] : false;
```

Comment:
Is the `!=` intentional vs `!==` ? Maybe `_.isNil` might be a good candidate?

Reply:
https://github.com/elastic/kibana/pull/34327#discussion_r272345067